ErgoScript-Developer-Course/batch-0/zuli/HW2_zuli/build.sbt

```diff
@@ -0,0 +1,84 @@
+import org.ergoplatform.compiler.ErgoScalaCompiler._
+import org.ergoplatform.playgroundenv.utils.ErgoScriptCompiler
+import org.ergoplatform.playground._
+
+//1. Booleans
+val thisbool = true
+val y = !false
+
+//2. if else statements
+val a=1
+val b=2
+val smallervalue = if (a < b ) a else b
+println(smallervalue)
+
+//Datatypes
+
+//long
+val x = 123L
+println(x)
+
+//Boolean
+val alreadytrue = thisbool
+println(alreadytrue)
+println(y)
+
+//Byte
+val something = Array[Byte](10, -20, 30)
+println(something)
+
+//complex data types
+
+//lists or collections
+val numbers = List(1,2,3,4)
+println(numbers)
+val fruit: List[String] = List("apples", "oranges", "pears")
+println(fruit)
+
+//pairs
+val ingredientamount = ("water" , 1000)
+println(ingredientamount._1)
+println(ingredientamount._2)
+
+//arbitrary data type
+var A:Map[Char,Int] = Map()
```

Comment:
In ErgoScript, you don't have access to a Map datastructure. Also, you can only use `val` not `var`, since variables are immutable only in ErgoScript. You could do something like this instead using Coll:
`val A: Coll[(Coll[Byte], Coll[Byte])]` where instead of map you make a collection of pairs, where the first element is the bytes representing the color name and the second element the bytes representing the string color code.